Relativity.API.md

```diff
@@ -0,0 +1,17 @@
+# Relativity.API Version Compatibility
+
+![Nuget](https://img.shields.io/nuget/v/Relativity.API)

```

Comment:
Mistake on my sample can you change this to - 
[![nuget](https://img.shields.io/nuget/v/Relativity.API.svg)](https://www.nuget.org/packages/Relativity.API)